### PR TITLE
Fin

### PR DESCRIPTION
**Pull request by danieljaffe**
We want the new versions where there are merge conflicts

### CODE REVIEW

File: server/src/graphql/api.ts [hunk 1]
```diff
@@ -86,7 +86,7 @@ export const graphqlRoot: Resolvers<Context> = {
       ctx.pubsub.publish('SURVEY_UPDATE_' + surveyId, survey)
       return survey
     },
-    addToQueue: async (_, { input }, {redis}) => {
+    addToQueue: async (_, { input }, {redis, pubsub}) => {
```
Comment (rfchang) [inline server/src/graphql/api.ts:92]: Great addition!

**Review by rfchang — APPROVED** [review]
Good code! Critical changes!

**Review by danieljaffe — COMMENTED** [review]
I dig it!

**Review by aybeshlikyan — APPROVED** [review]
Looks great! Ready to merge.

**Pull request merged**